submenu-flow.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Submenu Focus Flow Example</title>
+  <link rel="stylesheet" href="node_modules/normalize.css/normalize.css">
+  <link rel="stylesheet" href="css/base.css">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+</head>
+<body class="nav-list-example">
+
+  <header role="banner">
+    <h1>Managing focus with named anchors</h1>
+  </header>
+
+  <div class="flexContainer flexSpaceAround">
+    <div class="col">
+      <nav class="pf-c-vertical-nav">
+        <ul class="pf-c-vertical-nav__content" role="list">
+          <li class="pf-c-vertical-nav__item">
+            <a href="#" class="pf-c-vertical-nav__link">
+              <span class="pf-c-vertical-nav__link-icon"><i class="fas fa-home"></i></span>
+              <span class="pf-c-vertical-nav__link-text">Home</span>
+            </a>
+          </li>
+
+          <li class="pf-c-vertical-nav__item">
+            <a href="#" class="pf-c-vertical-nav__link">
+              <span class="pf-c-vertical-nav__link-icon"><i class="fas fa-home"></i></span>
+              <span class="pf-c-vertical-nav__link-text">Technology</span>
+            </a>
+          </li>
+
+          <li class="pf-c-vertical-nav__item">
+            <a
+              href="#navbarSubmenu01"
+              aria-expanded="false"
+              aria-current="false"
+              class="pf-c-vertical-nav__link"
+              id="navbarDropdownMenuLink01">
+              <span class="pf-c-vertical-nav__link-icon">
+                <i class="fas fa-home"></i>
+              </span>
+              <span class="pf-c-vertical-nav__link-text">Science</span>
+            </a>
+
+            <section
+              class="pf-vertical-sub-nav"
+              aria-hidden="true"
+              aria-labelledby="navbarDropdownMenuLink01"
+              id="navbarSubmenu01">
```

Comment:
I think we had determined with our [anchor link test](https://rawgit.com/andresgalante/jenn/master/focus-flow.html) that using anchor links don't work as well when the element is hidden at the time that you click the link. Can we split this element into two: one with the id that's always present, and the immediate child with aria-hidden to hide the contents? I know the css is funky in the example when we do this, but for this purpose, we're just testing if this method is a valid option to pursue.

Reply:
I also wonder if the `aria-labelledby` would be ineffective if that element is hidden at the time that the section receives focus. What are you thoughts on just changing this to `aria-label="Science"`, maybe even for just one of the submenus so that we can test if that is affected?

Reply:
Yes absolutely! Wanted to get that refactor out of the way first, I sort of forgot after all the shuffling that THIS was the PR that example wasn't ultimately supposed to be part of :) - working on it now

Reply:
@jgiardino for the "Tab flow for standard (hidden) submenu" and "Tab flow for visible submenu" demo, I adjusted the Science submenu to use `aria-label` directly as you suggested. It doesn't seem to make any difference for the standard hidden submenu, but I do see a difference in behavior for the visible submenu container. In VO it denotes the submenu as a region which shows up in the Landmarks rotor menu. Arrow key focus seems to be richer in this case. Would like to compare this to the JAWS experience.